<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFlightPlanDialog</class>
 <widget class="QDialog" name="CFlightPlanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flight plan</string>
  </property>
  <layout class="QVBoxLayout" name="vl_FlightPlan">
   <item>
    <widget class="QTextEdit" name="te_FlightPlan"/>
   </item>
   <item>
    <widget class="QWidget" name="wi_Buttons" native="true">
     <layout class="QHBoxLayout" name="hl_Buttons">
      <item>
       <widget class="swift::gui::components::CCallsignCompleter" name="comp_CallsignCompleter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pb_LoadFlightPlan">
        <property name="text">
         <string>load</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_FlightPlanButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CCallsignCompleter</class>
   <extends>QFrame</extends>
   <header>gui/components/callsigncompleter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bb_FlightPlanButtons</sender>
   <signal>accepted()</signal>
   <receiver>CFlightPlanDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_FlightPlanButtons</sender>
   <signal>rejected()</signal>
   <receiver>CFlightPlanDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
